10-std.tex

```diff
@@ -539,3 +539,145 @@ \section{Input/Output}
 
 \section{Sys/sys}
 \label{std-sys}
+
+\section{Remoting}
+\label{std-remoting}
+
+Haxe remoting is a way to talk between different platforms. With Haxe remoting we can communicate transparently, send data and call methods, between server and client side.
+
+\subsection{Remoting Connection}
+\label{std-remoting-connection}
+
+In order to use remoting, there must be a connection established first. There are two kinds of Haxe Remoting connections: 
```

Comment:
"an established connection", see if you take out the "first", since "in order to use..." already contains the meaning of "this must be done before anything else".